models/marts/web_events/fct_snowplow_sessions.sql

```diff
@@ -5,106 +5,90 @@
 }}
 
 with sessions as (
-    select * from  {{ ref('snowplow_sessions') }}
+    select * from  {{ ref('snowplow_sessions__with_channels') }}
 ),
 
 shopify_customers as (
+    
     select * from {{ ref('customers') }}
+    
 ),
 
-gclids as (
-    select * from {{ ref('snowplow_sessions_gclid') }}
-),
-
-join_adwords as (
+sessions_joined as (
 
     select
     
         sessions.*,
         date_trunc('day', cast(session_start as date)) as session_day,
-        shopify_customers.customer_created_at,
-        shopify_customers.customer_age_days,
-        shopify_customers.customer_type,
-        shopify_customers.lifetime_revenue,
-        shopify_customers.lifetime_completed_orders,
+        shopify_customers.customer_id,
```

Comment:
we need to be consistent about what "rules" the account - customer_id or email. then, all joins and partitions across the account should be on that field